selfupdate/detect.go

```diff
@@ -12,7 +12,7 @@ import (
 
```

Comment:
I did some refactorings yesterday. Could you rebase to master and resolve conflicts at first?

Reply:
It's rebased @rhysd 😰, I let you review the changes.